manifests/4.6/local-storage-operator.v4.6.0.clusterserviceversion.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: local-storage-operator.v4.6.0
```

Comment:
Why are we changing the 4.6 version when we also have the `local-storage-operator.v4.7.0.clusterserviceversion.yaml` ?

Reply:
err typo

Reply:
Fixed